src/Microsoft.ML.Data/Training/TrainerEstimatorBase.cs

```diff
@@ -62,6 +63,20 @@ public TrainerEstimatorBase(IHost host, SchemaShape.Column feature, SchemaShape.
             WeightColumn = weight;
         }
 
+        /// <summary>
+        /// Normal metadata that we produce for score columns.
+        /// </summary>
+        protected static IEnumerable<SchemaShape.Column> MetadataForScoreColumn(bool isNormalized = false)
```

Comment:
>isNormalized [](start = 85, length = 12)

will this come from the trainerInfo, or it is always true for the Propability type column?